dissect/target/helpers/config.py

```diff
@@ -1,23 +1,52 @@
-import importlib
+import ast
+import importlib.machinery
+import importlib.util
 from pathlib import Path
+from types import ModuleType
+from typing import Optional, Union
 
 CONFIG_NAME = ".targetcfg.py"
 
 
-def load(path):
+def load(path: Optional[Union[Path, str]]) -> ModuleType:
     config_spec = importlib.machinery.ModuleSpec("config", None)
     config = importlib.util.module_from_spec(config_spec)
     config_file = _find_config_file(path)
     if config_file:
-        exec(config_file.read_text(), config.__dict__)
+        config_values = _parse_ast(config_file.read_bytes())
+        config.__dict__.update(config_values)
     return config
 
 
-def _find_config_file(path):
+def _parse_ast(code: str) -> dict[str, Union[str, int]]:
+    # Only allow basic value assignments for backwards compatibility
+    obj = {}
+
+    module = ast.parse(code)
+    if not isinstance(module, ast.Module):
+        return obj
```

Comment:
Would it be an idea to have some debug logging here and at the `continue`s so you have some feedback when things don't parse?

Reply:
Added some.